script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -69,7 +69,7 @@ In the first place, there's no such thing as a 'line'; they're just a delusion t
 <Page17>
 "――――It's a lie."
 The gastric juice keeps dripping from my lips.
-Ignoring my mouth, my body from the chin down is sticky with stomach fluids.//　口はおろか、あごから下は胃液でベトベトだ。
+Not only my mouth, my body from the chin down is sticky with stomach fluids.//　口はおろか、あごから下は胃液でベトベトだ。
```

Comment:
feels like you need a `but` after the comma?
Not only my mouth, **but my entire body** from the chin down is sticky with stomach fluids.